library/src-bootstrapped/scala/internal/TypeBox.scala

```diff
@@ -0,0 +1,5 @@
+package scala.internal
+
+final abstract class TypeBox[-L, +U] {
```

Comment:
TypeBox can appear in error messages currently, e.g. with the original example from #4376
```scala
-- [E007] Type Mismatch Error: try/i4376.scala:5:5 -----------------------------
5 |    (Array.ofDim[String](1) : Array[_ >: Id[Nothing]])
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |Found:    Array[?1.CAP]
  |Required: Array[_ >: App.Id[_ <: Int]]
  |
  |where:    ?1 is an unknown value of type scala.internal.TypeBox[App.Id[Nothing], Any]
one error found
```
So we should either document what it means, or make the error message higher-level.

Reply:
TypeBox is gone now, fortunately.